.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04]
-        sourcemod-version: [1.11-dev]
+        sourcemod-version: [master]

```

Comment:
if we are using master then we need to pin to a specific commit instead:
- https://github.com/alliedmodders/sourcemod/commit/a402b3ccebbd85fe523774dad4864c71b3e87c6f
- a402b3ccebbd85fe523774dad4864c71b3e87c6f